packages/core/state/interaction/selectors.ts

```diff
@@ -174,17 +186,25 @@ export const getAnnotationService = createSelector(
         return new HttpAnnotationService({
             applicationVersion,
             userName,
+            aicsLoadBalancerBaseUrl: aicsLoadBalancerBaseUrl,
             baseUrl: fileExplorerBaseUrl,
         });
     }
 );
 
 export const getDatasetService = createSelector(
-    [getApplicationVersion, getUserName, getFileExplorerServiceBaseUrl, getRefreshKey],
-    (applicationVersion, userName, fileExplorerBaseUrl) =>
+    [
+        getApplicationVersion,
+        getUserName,
+        getAicsLoadBalancerBaseUrl,
+        getFileExplorerServiceBaseUrl,
+        getRefreshKey,
+    ],
+    (applicationVersion, userName, aicsLoadBalancerBaseUrl, fileExplorerBaseUrl) =>
         new DatasetService({
             applicationVersion,
             userName,
+            aicsLoadBalancerBaseUrl: aicsLoadBalancerBaseUrl,
```

Comment:
Is this actually used by the dataset service? If so, how?

Reply:
resolved in https://github.com/AllenInstitute/biofile-finder/pull/339